tests/Makefile

```diff
@@ -139,6 +139,9 @@ endif
 
 install-cosbench:
 	$(V)$(GIT) clone git@github.com:scality/zenko-cosbench.git build/zenko-cosbench && \
+	cd build/zenko-cosbench && \
+	git checkout f195f9ac57704b8923bde7fb6ed2095c2d95dc71 && \
```

Comment:
is it better to hard-code the commit hash like this or to set a var name for it?

Reply:
This is really just a quick fix until I get around to updating the test to use the new chart. Hopefully it won't hang around for long